common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -4,7 +4,7 @@
 
 <div class="edition-picker js-edition-picker-container">
 
-    <input class="edition-picker__checkbox js-enhance-checkbox"
+    <input class="edition-picker__button js-enhance-checkbox js-edition-picker-button"
```

Comment:
if so I think it should follow the BEM style `js-edition-picker__button` still (personal preference, don't think that's written down anywhere so up to you...)